assets/src/edit-story/elements/image.js

```diff
@@ -16,17 +16,18 @@ const Element = styled.img`
 	${ ElementWithRotation }
 `;
 
-function Image( { src, width, height, x, y, rotationAngle } ) {
+function Image( { src, width, height, x, y, rotationAngle, forwardedRef } ) {
 	const props = {
 		width,
 		height,
 		x,
 		y,
 		rotationAngle,
 		src,
+		ref: forwardedRef,
 	};
 	return (
-		<Element { ...props } />
+		<Element ref={ forwardedRef } { ...props } />
```

Comment:
Not needed both here and in `props`.

Reply:
The name of the variable is different here. Is that why this is done?

Reply:
```suggestion
		<Element { ...props } />
```
Not needed as not par of any other element.